testing/efl/APKBUILD

```diff
@@ -0,0 +1,72 @@
+# Contributor: Boris Faure <boris@fau.re>
+# Maintainer:
+pkgname=efl
+pkgver=1.21.1
+pkgrel=0
+pkgdesc="Enlightenment Foundation Libraries"
+url="https://www.enlightenment.org"
+arch="all"
+license="BSD-2 GPL-2 LGPL-2.1 ZLIB"
+depends="dbus fontconfig freetype fribidi giflib gstreamer gst-plugins-base libpng libjpeg-turbo libx11 libxcomposite libxcursor libxdamage libxinerama libxrandr libxtst libxscrnsaver luajit openssl"
+depends_dev="fontconfig-dev freetype-dev fribidi-dev giflib-dev gstreamer-dev gst-plugins-base-dev libpng-dev libjpeg-turbo-dev libx11-dev libxcomposite-dev libxcursor-dev libxdamage-dev libxinerama-dev libxrandr-dev libxtst-dev libxscrnsaver-dev luajit-dev openssl-dev xorgproto"
+makedepends="$depends_dev dbus-dev"
+install=""
+subpackages="$pkgname-dev"
+source="https://download.enlightenment.org/rel/libs/$pkgname/$pkgname-$pkgver.tar.xz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--localstatedir=/var \
+                --enable-image-loader-png \
```

Comment:
The indentation is inconsistent here and a few other lines.

Reply:
I should have fixed those.